ecg/jgi.py

```diff
@@ -109,75 +111,319 @@ def __get_domain_json(self,domain_url,database,domain):
         domain_json = json.loads(self.driver.find_element_by_tag_name('body').text)
         return domain_json
 
-    def __get_organism_urls(self,domain_json):
+    def __get_organism_urls(self,domain_json,organism_url_path):
 
         all_GenomeNameSampleNameDisp =  [d['GenomeNameSampleNameDisp'] for d in domain_json['records']]
-
         organism_urls = list()
 
         for htmlandjunk in all_GenomeNameSampleNameDisp:
             regex = r"<a href='main\.cgi(.*)'>"
             match = re.search(regex, htmlandjunk)
             html_suffix = match.group(1)
-            full_url = self.homepage_url+html_suffix
+            
+            #full_url = self.homepage_url+html_suffix
+            full_url = "https://img.jgi.doe.gov/cgi-bin/m/main.cgi{}".format(html_suffix)
```

Comment:
Can you change this to reference `self.homepage_url+"{}"`?

Reply:
Sure.